src/argowrapper/auth/utils.py

```diff
@@ -0,0 +1,24 @@
+import requests
+from argowrapper import logger
+
+
+def get_cohort_ids_for_team_project(token, source_id, team_project):
+    header = {"Authorization": token, "cookie": "fence={}".format(token)}
+    url = "http://cohort-middleware-service/cohortdefinition-stats/by-source-id/{}/by-team-project?team-project={}".format(
```

Comment:
This should probably be a configurable url - at the very least so no one raises an unnecessary stink about a non-https request grabbing identifiers used for access control logic.